content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -682,6 +682,190 @@ The new Job itself will have a different uid from `a8f3d00d-c6d2-11e5-9f87-42010
 `manualSelector: true` tells the system that you know what you are doing and to allow this
 mismatch.
 
+### Pod failure policy {#pod-failure-policy}
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+{{< note >}}
+You can only configure a Pod failure policy for a Job if you have the
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+enabled in your cluster: `JobPodFailurePolicy`. Additionally, it is recommended
+to enable the `PodDisruptionsCondition` feature gate in order to be able to detect and handle
+pod disruption conditions in the Pod failure policy (see also:
+[Pod disruption conditions](/docs/concepts/workloads/pods/disruptions#pod-disruption-conditions)). Both feature gates are
+available in Kubernetes v1.25.
+{{< /note >}}
+
+In some situations you may want to have a better control over handling of Pod
+failures than the default policy based on the Job's `.spec.backoffLimit` gives
+(see: [Pod backoff failure policy](#pod-backoff-failure-policy)).
+
+For example, you may want to terminate a job as soon as one of its Pods fails
+with an exit code indicating a software bug. This may allow you to cut down costs of running
+workloads by avoiding unnecessary Pod restarts. On the other hand, you may want
+not to count pod failures caused by disruptions (such as preemption, eviction API
+or taint-based Pod deletions) towards the `.spec.backoffLimit` counter.
+
+You can satisfy the above use cases by specifying a Pod failure policy
+(in the `.spec.podFailurePolicy` field) for handling Pod failures based on the
+container exit codes and the Pod conditions.
+
```

Comment:
I agree that a list would help understand the 2 use cases. However, this is not the right way to cut them.

```markdown
In some situations, you  may want to have a better control when handling Pod
	failures than the control provided by the default policy, which is based on the Job's (`.spec.backoffLimit`](#pod-backoff-failure-policy)). These are some examples of use cases:

* To optimize costs of running
	workloads by avoiding unnecessary Pod restarts, you can terminate a job as soon as one of its Pods fails with an exit code indicating a software bug. 
* To guarantee that your job finishes even if there are disruptions, you can ignore pod failures caused by disruptions (such as preemption, eviction API,
	or taint-based Pod deletions) so that they don't count towards the `.spec.backoffLimit` counter.
```

Reply:
Applied, but rephrased a bit in the latest commit, please take a look

Reply:
looks good :smiley: